src/components/NewWorkspaceModal.ts

```diff
@@ -318,7 +319,17 @@ const NewWorkspaceModal = withDisplayName(
       // This coupling of enhanced bucket logging and billing project may change in the future.
       return Utils.cond(
         [!!workflowImport || !!requireEnhancedBucketLogging, () => !isAzureBillingProject(project)],
-        [!!cloneWorkspace && isAzureWorkspace(cloneWorkspace), () => isAzureBillingProject(project)],
+        [
+          !!cloneWorkspace && isAzureWorkspace(cloneWorkspace),
+          () => {
+            if (!isAzureBillingProject(project)) {
+              return false;
+            }
+            const azureBillingProject = project as AzureBillingProject;
```

Comment:
@msilva-broad is there a better way to do this? I would have hoped that the check on line 325 would allow TS to know that the billing project is an Azure one.

Reply:
Thanks, fixed.